Mathlib/RingTheory/OreLocalization/Basic.lean

```diff
@@ -942,3 +942,6 @@ instance divisionRing : DivisionRing R[R⁰⁻¹] :=
 end DivisionRing
 
 end OreLocalization
+
+open Lean
+#check Lean.Expr.const `a []
```

Comment:
should this be here or in a separate test file? I guess at least a comment would be nice